app/Litteraturkritikk/Publications.php

```diff
@@ -2,11 +2,9 @@
 
 namespace App\Litteraturkritikk;
 
-use Illuminate\Support\Arr;
-
-class Publications {
-
-    static protected $newspapers = [
+class Publications
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class Publications has 1249 lines of code. Current threshold is 1000. Avoid really long classes.](https://app.codacy.com/app/scriptotek/ub-baser/pullRequest?prid=4643802)